### PR TITLE
CREST-7: cleaning up examples

### PR DESCRIPTION
**Pull request by kyongkim-illumio**
Cleaning up some examples. User reported incorrect source and version. Also found an issue with ip_list example.

### CODE REVIEW

File: examples/ip_list/main.tf [hunk 1]
```diff
@@ -21,7 +21,8 @@ resource "illumio-core_ip_list" "example" {
   description = "example desc"
   ip_ranges {
     from_ip = "1.1.0.0/24"
-    to_ip = "0.0.0.0/0"
+    // from_ip = "1.1.0.0"
+    // to_ip = "1.1.0.254"
```
Comment (dsommerville-illumio) [inline examples/ip_list/main.tf:25]: Why are these commented out? Are we not sure what the values should be here?

**Review by dsommerville-illumio — CHANGES_REQUESTED** [review]
LGTM, just one small nit.

We should clean up these examples as this amount seems a bit unmanageable. What are the most important endpoints?

File: examples/ip_list/main.tf (hunk 1 above)
Comment (kyongkim-illumio) [reply examples/ip_list/main.tf:25]: I wanted to show both examples of CIDR for from_ip and individual ip for from_ip and to_ip. I can remove it if it makes it look sloppy.

Comment (dsommerville-illumio) [reply examples/ip_list/main.tf:25]: That makes sense - can you add a comment to explain that above?

**Review by dsommerville-illumio — APPROVED** [review]
Looks good, thanks!

**Pull request merged**